### PR TITLE
Advancements till the first GSoC review

### PR DESCRIPTION
**Pull request by Cheesetouched**

### CODE REVIEW

File: compute/do-droplets.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      })
+    });
+  }
+
+  /**
+   * List all available Kernels for a Droplet
+   * @listKernels
+   * @param {string} id
+   */
+  listKernels(id) {
```
Comment (rajikaimal) [inline compute/do-droplets.js]: What's the equivalent function in aws-plugin?

Comment (Cheesetouched) [reply compute/do-droplets.js]: There's a way to connect a pre-created Linux instance using SSH. But that can be achieved only using a manual series of steps and not through a delegated API or SDK, which is in contrast to what DigitalOcean provides.

TL;DR We can't achieve the same using aws-plugin

File: do.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   */
+  LoadBalancers() {
+    return new LoadBalancers(this.getSDK(), this.getToken());
+  }
+
+  /**
+   * Networking - Floating IPs Wrapper
+   * @FloatingIps
+   */
+  FloatingIps() {
```
Comment (rajikaimal) [inline do.js]: Change method names to camelCase

Comment (Cheesetouched) [reply do.js]: I've followed camelCase throughout the codebase with the wrapper classes as an exception. These method names belong to the wrapper classes. Should we necessarily also change them to camelCase?

**Comment by Cheesetouched** [comment]
Hey @rajikaimal, I've added camelCase as you requested. Please review the updated codebase.

File: compute/do-droplets.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  /**
+   * List snapshots for a Droplet
+   * @listSnapshots
+   * @param {string} id
+   */
+  listSnapshots(id) {
+    return new Promise((resolve,reject) => {
+      this._client.droplets.snapshots(id,(err,result) => {
+        if (err) reject(err)
+        console.log(result)
```
Comment (rajikaimal) [inline compute/do-droplets.js]: Let's remove all logs

Comment (Cheesetouched) [reply compute/do-droplets.js]: Do you mean all the console.logs?

Comment (rajikaimal) [reply compute/do-droplets.js]: Yes

**Comment by Cheesetouched** [comment]
Hey @rajikaimal, I've removed all the console.logs from the wrapper classes. Please review the updated codebase.

**Pull request merged**